docs/learn/encyclopedia/storage/state-archival.mdx

```diff
@@ -69,6 +69,14 @@ A call to `extend_ttl(N)` ensures that the current TTL of the contract instance
 
 In addition to contract-defined TTL extensions using the `extend_ttl()` function, a contract data entry's TTL can be extended via the [`ExtendFootprintTTLOp`](#extendfootprintttlop) operation.
 
+## Behavior of transactions that try to access an archived Persistent entry
+
+Here are some important points to keep in mind when it comes to archived entries -
+
+1. A Soroban transaction that has a key to an archived Persistent entry in the footprint will fail immediately during the apply stage prior to contract execution. It does not matter if the contract itself was going to access the entry.
+2. Due to the previous point that archived entries can never make it into smart contract logic, there is no reason to write code in your contract to handle archived entries. The same applies to contract test cases - while you may want to write tests that check if your extension logic is correct, you don't need to write archival tests because the transaction will fail before getting to the contract.
```

Comment:
We could refer to https://developers.stellar.org/docs/build/guides/archival/test-ttl-extension for the TTL testing examples (it's actually possible to observe the env panic when accessing an archived entry, though I don't think that's necessary to promote this pattern)

Reply:
Good point. Done.